cmd/serverless/main.go

```diff
@@ -201,8 +201,11 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 		log.Error("No API key configured, exiting")
 	}
 	config.Datadog.SetConfigFile(datadogConfigPath)
+	// Load datadog.yaml file into the config, so that metricAgent can pick these configurations
+	if _, err := config.Load(); err == nil {
+		log.Error("Error happened when loading configuration from datadog.yaml for metric agent.")
```

Comment:
```suggestion
		log.Error("Error happened when loading configuration from datadog.yaml for metric agent: %w", err)
```

Might be helpful to include the original error in the message here. Though I'm pretty it already gets logged from within `config.Load()`, might be worth repeating it here.

Reply:
Changed in the new revision